### PR TITLE
📝 Update README in Chinese

### PR DESCRIPTION
**Pull request by AlexV525**
# Description

Update README in Chinese. Following the style guide that Chinese documents usually use. Few things were updated in this change:
* Words correct/improve.
* Remove a user group link since it's nothing official.

The typography guide is based on [中文文案排版指南 (Chinese Typography Guide)](https://github.com/cfug/flutter.cn/wiki/%E4%B8%AD%E6%96%87%E6%96%87%E6%A1%88%E6%8E%92%E7%89%88%E6%8C%87%E5%8D%97-(Chinese-Typography-Guide)) .

### Short Bio

A maintainer in the [CFUG (Chinese Flutter User Group)](https://github.com/cfug) Localization team. Reviewed over 200 articles translation in Chinese. Also a Flutter GitHub member.

Please let me know if the doc needs update in the future, and I'm glad I can help with it.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors and are passing (See [Contributor Guide]).
- [x] My PR does not decrease the code coverage, or I have __a very special case__ and explained on the PR description why this PR decreases the coverage.
- [x] I updated/added relevant documentation (doc comments with `///`) and updated/added examples in `doc/examples`.
- [x] I have formatted my code with `./scripts/format.sh` and the Flame analyzer (`./scripts/analyze.sh`) does not report any problems.
- [x] I read and followed the [Flame Style Guide].
- [x] I have added a description of the change under `[next]` in `CHANGELOG.md`.
- [x] I removed the `Draft` status, by clicking on the `Ready for review` button in this PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flame users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in `CHANGELOG.md`).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flame-engine/flame/issues
[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Flame Style Guide]: https://github.com/flame-engine/flame/blob/main/STYLEGUIDE.md

### CODE REVIEW

**Comment by spydon** [comment]
Hi, thanks for your contribution! Is this updated to be aligned with the english readme too?

**Comment by AlexV525** [comment]
Yes, was aligned with the latest official version.

**Comment by spydon** [comment]
@HarrisonQi could you review this?

**Comment by AlexV525** [comment]
Well I think he's AFK for a while. We can push this forward.

**Comment by HarrisonQi** [comment]
> @HarrisonQi could you review this?

Yes, I can. Just wait a moment

**Comment by HarrisonQi** [comment]
We should keep the communication platform provided for users from China, even it is not official. Users from China cannot use discord directly. Current document has been marked as unofficial.

**Comment by AlexV525** [comment]
Thanks and good to hear your feedback. IMO if we need to add some groups link or any unofficial contents, at least a warning should be presented. I didn't see any tips mentioned that this is in unofficial README, can you point it out?

**Comment by AlexV525** [comment]
If you need to discuss with this PR more specifically, you can [add me](http://wpa.qq.com/msgrd?v=3&uin=505646850&site=qq&menu=yes) with QQ.

File: i18n/README-ZH.md [hunk 1]
```diff
@@ -21,73 +20,78 @@
...
-顺手点一颗Star, 就可以提供你的帮助!
+顺手点一颗 Star，就是你对我们的帮助！
 
-你也可以成为赞助者, 通过Patreon来支持我们:
+你也可以成为赞助者，通过 Patreon 来支持我们：
 
 [![Patreon](https://c5.patreon.com/external/logo/become_a_patron_button.png)](https://www.patreon.com/fireslime)
 
-或者通过买杯咖啡捐赠我们:
+或者通过 But Me A Coffet 捐赠我们：
```
Comment (HarrisonQi) [inline i18n/README-ZH.md]: Maybe it's `Buy Me a Coffe`, not `But Me A Coffet`.

File: i18n/README-ZH.md [hunk 2]
```diff
@@ -101,28 +105,31 @@ Flame的官方网站(其中包含文档), 请[点击此处](https://flame-engine
...
 
-如果你找到了bug或者有使Flame更好的建议, 创建一个issue. 我们将会尽快处理.
+如果你找到了 bug 或对 Flame 有更好的建议，请创建一个 issue，我们将会尽快处理。
 
-你想通过PR做贡献吗? PR总是受欢迎的, 只要确保你的分支是`develop`, 并遵循[PR模板](.github/pull_request_template.md)即可.
+你想通过 PR 进行贡献吗？我们欢迎 PR，只要确保你的分支是 `develop`，并遵循 [PR 模板](.github/pull_request_template.md) 即可。
 
 ## 立即开始
 
-简体中文教程 [Alekhin](https://github.com/japalekhin) Tutorial Series (Simplified Chinese) by [HarrisonQI](https://github.com/HarrisonQi)
```
Comment (HarrisonQi) [inline i18n/README-ZH.md:114]: We should keep those articles. There's so many Chinese users because those translated articles.

**Review by HarrisonQi — COMMENTED** [review]
> Thanks and good to hear your feedback. IMO if we need to add some groups link or any unofficial contents, at least a warning should be presented. I didn't see any tips mentioned that this is an unofficial README, can you point it out?


Sure we mentined this at line 28:
```
> 本篇为社区版翻译, 目前针对版本: 0.29.0
```

When I first wrote this readme, Flame team reminded this for the first time.

File: i18n/README-ZH.md (hunk 1 above)
Comment (AlexV525) [reply i18n/README-ZH.md]: It's been updated.

File: i18n/README-ZH.md (hunk 2 above)
Comment (AlexV525) [reply i18n/README-ZH.md:114]: Could you provide a repo link instead of a user profile link?

**Comment by spydon** [comment]
@AlexV525 what about we keep all your updates but also keep the link to the unofficial chat in there since people in China can't access discord? @HarrisonQi received permission earlier to have that link in the readme. Seems like a waste to throw away your contribution @AlexV525? :) 

On the same topic, it shouldn't be "more flexible content" in there, it should be a translation of the official readme. But keeping it up to date is of course a best effort since everything is done on a free-time basis.

**Comment by AlexV525** [comment]
> @AlexV525 what about we keep all your updates but also keep the link to the unofficial chat in there since people in China can't access discord? @HarrisonQi received permission earlier to have that link in the readme. Seems like a waste to throw away your contribution @AlexV525? :)

I'm okay with your advice, but I would wait for @HarrisonQi 's opinion since he first put effort into this translation.

**Comment by HarrisonQi** [comment]
We'll update a new readme(Chinese) after v1.0.0. If there are not too many changes, we will learn part of translations from @AlexV525.

**Comment by HarrisonQi** [comment]
When we have completed all the document translation, we will upload the document to the github warehouse and gradually replace it with the official link. 

The translation work may take several months. So until we provide a complete translation, some of our documents cannot be directly linked to the official link

**Comment by spydon** [comment]
@HarrisonQi what documents are you talking about? It should be possible to update this readme without having a dependency on other documents.

**Comment by HarrisonQi** [comment]
Except for the removal of part of the Chinese communication portal, this PR almost no changes(Maybe adjusted part of the grammar).

File: i18n/README-ZH.md (hunk 2 above)
Comment (HarrisonQi) [reply i18n/README-ZH.md:114]: Ok, you may remove. We'll update these articles to flame-cn.com

**Comment by AlexV525** [comment]
Again, discussed with @HarrisonQi offline, and the conclusion is the group link will stay. Other links will be temporarily removed, until it got published in the official localization site.

**Comment by HarrisonQi** [comment]
LGTM

**Comment by spydon** [comment]
@HarrisonQi @AlexV525 great that you came to an agreement, let's merge this! :)

**Review by spydon — APPROVED** [review]

**Pull request merged**